html/includes/forms/delete-host-dependency.inc.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * LibreNMS
+ *
+ * Copyright (c) 2017 Aldemir Akpinar <https://github.com/aldemira/>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+header('Content-type: text/plain');
```

Comment:
We should definitely switch this to output json (some of the earlier form code I did was plain text and it was wrong / bad so don't base it on that).  See this PR for examples: https://github.com/librenms/librenms/pull/7489/files